tobac/__init__.py

```diff
@@ -29,47 +29,47 @@
     plot_mask_cell_track_follow,
     plot_mask_cell_track_static,
     plot_mask_cell_track_static_timeseries,
-)
-from .plotting import (
     plot_lifetime_histogram,
     plot_lifetime_histogram_bar,
     plot_histogram_cellwise,
     plot_histogram_featurewise,
-)
-from .plotting import plot_mask_cell_track_3Dstatic, plot_mask_cell_track_2D3Dstatic
-from .plotting import (
+    plot_mask_cell_track_3Dstatic,
+    plot_mask_cell_track_2D3Dstatic,
     plot_mask_cell_individual_static,
     plot_mask_cell_individual_3Dstatic,
+    animation_mask_field,
+    make_map,
+    map_tracks,
 )
-from .plotting import animation_mask_field
-from .plotting import make_map, map_tracks
-from .analysis import (
+from .analysis.analysis import (
```

Comment:
@w-k-jones : Nice work!

Two suggestions:
- I don't really like analysis.analysis . It should be more specific. What about renaming it to analysis/statistics.py?
- I would recommend to `analysis/__init__.py` the following lines:
```python
from .analysis import *
from .spatial import *
```
of after rneaming
```python
from .statistics import *
from .spatial import *
```
- it is already the case that refactoring has broken things in the notebook `tobac/examples/Basics/Idealized-Case-1_Tracking-of-a-Test-Blob-in-2D.ipynb` --> Better the keep the old structure for a while.

Reply:
I've split up the `analysis.analysis` module into `cell_analysis` and `feature_analysis` to separate them a bit better, but I'm still not sure what a better name for them is. I didn't want to use `statistics` to avoid confusion with the `bulk_statistics` module! I've added `__all__` statements to both and added imports to the `__init__.py` file to ensure back-compatibility